src/accordion/stories/Accordion.tsx

```diff
@@ -0,0 +1,200 @@
+import * as React from "react";
```

Comment:
Can we rename this file to AccordionComponent.ts ?? since it's confusing because we have the Accordion.ts too

Reply:
I have done the same to Meter & other files I have worked on.
I will change them in the next PR which I forked from this branch

Reply:
I have already made changes to the same file, changing now and merging will give extra job.